### DISCUSSION TITLE
rAudio 20230101: Error at update (mpdconf)

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1179] [Fixed Issues]
Updating 202212291 to 20230101 generates error:
"mv: inter-device move failed: '/tmp/mpdconf' to '/srv/http/data/mpdconf'; unable to remove target: Directory not empty"
Consequence: Missing custom settings in player window

**Comment by ceejayemm** [comment]
I can confirm this issue during update to 20230101. After update completes (?) I can see:

rAudio:~  #  ls -lrt /srv/http/data/mpdconf                                                        
total 16
drwxr-xr-x 2 http http 4096 Nov 18 05:56 conf
lrwxrwxrwx 1 http http   39 Dec 17 17:05 ffmpeg.conf -> /srv/http/data/mpdconf/conf/ffmpeg.conf
-rwxr-xr-x 1 http http  760 Dec 29 16:13 mpd.conf
-rwxr-xr-x 1 http http   26 Dec 29 16:13 curl.conf
-rwxr-xr-x 1 http http  224 Jan  1 11:08 output.conf

The link to ffmpeg.conf is presumably because I used SharedData ?

Chris
